survey/survey_app/models.py

```diff
@@ -24,11 +30,18 @@ class Option(models.Model):
     def __str__(self):
         return self.text
 
+class AnswerManager(models.Manager):
+    def user_submission_count_for_day(self, user,date):
+        today = timezone.now().date()
+        return self.filter(creator=user, created_at__date=today).count()
 
 class Answer(models.Model):
     question = models.ForeignKey(Question, on_delete=models.CASCADE, default=None)
     option = models.ForeignKey(Option, on_delete=models.CASCADE)
     creator = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
+    created_at = models.DateTimeField(default=timezone.now)
```

Comment:
Another solution is you can inherit from `TimeStampedModel` : https://dev.to/hossein761/make-your-django-models-self-timestamp-3i8e and do not need to implement `created_at` your own.

Reply:
ok